src/components/Modal/Modal.js

```diff
@@ -36,10 +43,21 @@ export default class Modal extends Component {
     iconDescription: 'close the modal',
     modalHeading: '',
     modalLabel: '',
+    selectorsFloatingMenus: ['.bx--overflow-menu-options__btn'],
```

Comment:
Do you think it's possible to align the list with vanilla? https://github.com/IBM/carbon-components/blob/3e9bc26/src/components/modal/modal.js#L194